<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KeyBrdReflectionWindowClass</class>
 <widget class="QMainWindow" name="KeyBrdReflectionWindowClass">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>660</width>
    <height>420</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>660</width>
    <height>420</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>KeyBrdReflectionWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="0">
     <layout class="QVBoxLayout" name="TextBoxLayout">
      <property name="leftMargin">
       <number>30</number>
      </property>
      <property name="topMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>30</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QTextEdit" name="BOX_ShowContent">
        <property name="font">
         <font>
          <family>Consolas</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="acceptDrops">
         <bool>false</bool>
        </property>
        <property name="lineWrapColumnOrWidth">
         <number>-1</number>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="acceptRichText">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="3" column="0">
     <layout class="QHBoxLayout" name="TimeInputLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>30</number>
      </property>
      <property name="topMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>30</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QCheckBox" name="STC_KeepOnTop">
        <property name="text">
         <string>窗口始终置顶</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <property name="autoExclusive">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="BTN_WriteConfirm">
        <property name="text">
         <string>准备写入</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="FileSelectionLayout">
      <property name="leftMargin">
       <number>30</number>
      </property>
      <property name="topMargin">
       <number>15</number>
      </property>
      <property name="rightMargin">
       <number>30</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="LAB_TIT_FilePath">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>文件路径：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="BOX_ShowPath">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="BTN_RefreshContent">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>23</width>
          <height>23</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>300</width>
          <height>300</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset theme="QIcon::ThemeIcon::ViewRefresh"/>
        </property>
        <property name="iconSize">
         <size>
          <width>13</width>
          <height>13</height>
         </size>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="BTN_OpenTargetFile">
        <property name="text">
         <string>打开文件...</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="BoxTitleLayout">
      <property name="leftMargin">
       <number>30</number>
      </property>
      <property name="topMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>30</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="LAB_TIT_ShowFileContentTitle">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>文件内容：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="LAB_ShowFileWordCount">
        <property name="text">
         <string>TextLabel</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="KeyBrdReflectionWindow.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>BTN_OpenTargetFile</sender>
   <signal>clicked()</signal>
   <receiver>KeyBrdReflectionWindowClass</receiver>
   <slot>GetTargetFileName()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>620</x>
     <y>46</y>
    </hint>
    <hint type="destinationlabel">
     <x>599</x>
     <y>70</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>BTN_WriteConfirm</sender>
   <signal>clicked()</signal>
   <receiver>KeyBrdReflectionWindowClass</receiver>
   <slot>StartButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>620</x>
     <y>387</y>
    </hint>
    <hint type="destinationlabel">
     <x>577</x>
     <y>307</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>STC_KeepOnTop</sender>
   <signal>checkStateChanged(Qt::CheckState)</signal>
   <receiver>KeyBrdReflectionWindowClass</receiver>
   <slot>SwitchWindowOnTop()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>134</x>
     <y>385</y>
    </hint>
    <hint type="destinationlabel">
     <x>234</x>
     <y>339</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>BTN_RefreshContent</sender>
   <signal>clicked()</signal>
   <receiver>KeyBrdReflectionWindowClass</receiver>
   <slot>RefreshFileContent()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>526</x>
     <y>37</y>
    </hint>
    <hint type="destinationlabel">
     <x>363</x>
     <y>9</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>GetTargetFileName()</slot>
  <slot>StartButtonClicked()</slot>
  <slot>RefreshFileContent()</slot>
  <slot>SwitchWindowOnTop()</slot>
 </slots>
</ui>
